envoy.yaml

```diff
@@ -0,0 +1,76 @@
+static_resources:
```

Comment:
better move these file into an new directory? i.e. "example"?

Reply:
That sounds great.

Reply:
this file should be removed? since it's already under example.